src/main/webapp/help/parasoftCoverage/help-baseline.html

```diff
@@ -21,10 +21,10 @@
             will focus on new or modified code only.
         </dd>
         <br/>
-        <dt>Modified code lines (difference to overall project) - MODIFIED_LINES_DELTA</dt>
+        <dt>Modified code lines (difference to modified files) - MODIFIED_LINES_DELTA</dt>
         <dd>
-            Difference between the project coverage and the modified lines coverage of the current build. Teams can use this delta
-            value to ensure that the coverage of pull requests is better than the whole project coverage.
+            Difference between the modified files coverage and the modified lines coverage of the current build. Teams can use this delta
+            value to ensure that the coverage of pull requests is better than the modified files coverage.
```

Comment:
Teams can use this delta value to compare between the modified lines coverage and the modified files coverage in pull requests.